src/scripts/main.js

```diff
@@ -1,3 +1,409 @@
 'use strict';
 
-// write your code here
+// Game state variables
+let gameGrid = [];
+let score = 0;
+let gameOver = false;
+let win = false;
+
+// DOM elements
```

Comment:
Remove comments everywhere. Try it write obvious code that explain all we need without comments)